### ISSUE TITLE
[Private Browsing] Background colour is incorrect for settings and library screens

### ISSUE THREAD

**Issue by AmyYLee** [mozilla-mobile/fenix#8492]
Steps to reproduce: 

1. Go to Settings
2. Go to Library (Bookmarks, History)

Expected background colour #1D1133
Current background colour: #291D4F

Right now these screens are using the PBM menu colour so the menus blend into the background and it's hard to see. 

![Screenshot_20200218-131212](https://user-images.githubusercontent.com/5992746/74764808-6c88fe00-5250-11ea-86d4-fb288efbb07a.png)
![Screenshot_20200218-130715](https://user-images.githubusercontent.com/5992746/74764811-6dba2b00-5250-11ea-92ea-2b7352d7f9b8.png)

**Comment by BranescuMihai** [comment]
@AmyYLee I looked into it, and the current state is:
- fragment background - #261E4B
- menu background - #291D4F

The issue is that the background colour(foundation) is the same for all fragments(except Home), as well as some other components. Do you want us to change only for Settings and History/Bookmarks, or for all of them?

For us I think it's much easier to change the colour of the menus instead, what do you think about this?

Side Note: is there a document with the colour palette used in our app? I see that we have about 10 shades of purple, and it's getting a bit hard to follow in code.

**Comment by AmyYLee** [comment]
> @AmyYLee I looked into it, and the current state is:
> 
>     * fragment background - #261E4B
> 
>     * menu background - #291D4F
> 
> 
> The issue is that the background colour(foundation) is the same for all fragments(except Home), as well as some other components. Do you want us to change only for Settings and History/Bookmarks, or for all of them?
> 
> For us I think it's much easier to change the colour of the menus instead, what do you think about this?
> 
> Side Note: is there a document with the colour palette used in our app? I see that we have about 10 shades of purple, and it's getting a bit hard to follow in code.

@BranescuMihai Hi, the settings background colour should apply to History/Bookmarks as well. This is the guideline for what colours should be used. I don't think we should change the menu colour as we are trying to keep consistency with the order of colours  being applied to each layer of the UI (dark shades to light).

Here's a guide for reference. 
https://share.goabstract.com/a469d33c-f5ef-4d0c-aa8a-bccf4ce324e3?mode=design&sha=3ad6799352105a533c0b5740502c346d1364e3cd
